internal/zinc-persist/src/main/scala/xsbt/api/CompanionsFormat.scala

```diff
@@ -0,0 +1,16 @@
+package xsbt.api
+
+import xsbti.api._
+import sbinary._
+
+object CompanionsFormat extends Format[Companions] {
```

Comment:
If we move all compatnions to single store then we can serialized whole map not single companion one by one. I think we should let Java optimized our serialization